core/src/main/java/com/graphhopper/util/PathMerger.java

```diff
@@ -99,8 +99,6 @@ public void doWork(PathWrapper altRsp, List<Path> paths, Translation tr) {
                     if (pathIndex + 1 < paths.size()) {
                         ViaInstruction newInstr = new ViaInstruction(fullInstructions.get(fullInstructions.size() - 1));
                         newInstr.setViaCount(pathIndex + 1);
```

Comment:
can we remove setViaCount as getViaCount is unused?

Reply:
> can we remove setViaCount as getViaCount is unused?

The viaCount is used in the `getTurnDescription` mehtod of the ViaInstruction.

Reply:
Ah, ok. Makes sense. Is it possible to move this to the constructor?